frontend/online-shop/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
This test is probably not working anymore so you can remove this file

Reply:
Fixed.